### PR TITLE
fix(atomic): show facet search results on consecutive searches

### PR DESCRIPTION
**Pull request by nrawji**
KIT-546
https://coveord.atlassian.net/browse/KIT-546

### CODE REVIEW

File: packages/atomic/src/components/facets/facet-search/facet-search.tsx [hunk 1]
```diff
@@ -223,6 +225,7 @@ export class FacetSearch {
   }
 
   public render() {
+    console.log(this.props.controller.facetSearchState);
```
Comment (samisayegh) [inline packages/atomic/src/components/facets/facet-search/facet-search.tsx:228]: Let's remove this log?

File: packages/atomic/src/components/facets/facet-search/facet-search.tsx [hunk 2]
```diff
@@ -89,6 +90,7 @@ export class FacetSearch {
     this.props.controller.facetSearch.select(
       value as CategoryFacetSearchResult
     );
+    this.props.controller.text = '';
```
Comment (samisayegh) [inline packages/atomic/src/components/facets/facet-search/facet-search.tsx:93]: Should we include cypress tests in this PR? Sometimes I see we are writing them, sometimes it is QA, so not sure?

**Review by samisayegh — APPROVED** [review]

File: packages/atomic/src/components/facets/facet-search/facet-search.tsx (hunk 2 above)
Comment (ThibodeauJF) [reply packages/atomic/src/components/facets/facet-search/facet-search.tsx:93]: Might have slipped by me before, but why is the controller's state modified directly here, in the FacetSearchController:

```
public set text(text: string) {
  -> this.state.facetSearchQuery = text; 
  this.facetSearch.updateText(text);
  this.facetSearch.search();
}
```
Is there an action missing on the headless controller?

Comment (ThibodeauJF) [reply packages/atomic/src/components/facets/facet-search/facet-search.tsx:93]: Ohh I was confused, it's actually a state in Stencil, makes sense now 😄

**Review by ThibodeauJF — APPROVED** [review]

**Review by olamothe — COMMENTED** [review]
We want an accompanying test for this in Cypress, especially for bugfix like that.

**Pull request merged**